diff --git a/src/common_func/common_func.hpp b/src/common_func/common_func.hpp
--- a/src/common_func/common_func.hpp
+++ b/src/common_func/common_func.hpp
@@ -3,8 +3,8 @@
 #include <iostream>
 
 
-#define SLIC3R_APP_NAME "Snapmaker Orca"
-#define SLIC3R_APP_KEY "Snapmaker_Orca"
+#define SLIC3R_APP_NAME "DuckySlicer"
+#define SLIC3R_APP_KEY "DuckySlicer"
 #define SLIC3R_VERSION "01.10.01.50"
 #define Snapmaker_VERSION "2.3.3"
 #define MIN_FIRM_VER "1.0.0"
