diff --git a/src/libslic3r/Format/3mf.cpp b/src/libslic3r/Format/3mf.cpp
--- a/src/libslic3r/Format/3mf.cpp
+++ b/src/libslic3r/Format/3mf.cpp
@@ -797,5 +797,10 @@
                 }
                 */
+                if (boost::algorithm::iequals(name, LAYER_CONFIG_RANGES_FILE)) {
+                    _extract_layer_config_ranges_from_archive(archive, stat, config_substitutions);
+                    continue;
+                }
+
                 // only read the model config for Prusa 3mf
                 if (boost::algorithm::iequals(name, MODEL_CONFIG_FILE)) {
                     // extract slic3r model config file
